### ISSUE TITLE
Is it possible to recognize text from specific region?

### ISSUE THREAD

**Issue by KlossHans** [rainmanwy/robotframework-SikuliLibrary#61]
I would like to recognize text from specific part of screen. Is it possible in SikuliLibrary to define region (startX, startY, width, height)?
I know that there is Get Text keyword where you can put image as parameter. But to be honest I don't understand idea of getting text from region matched by image.... Is it intended to be executed with low "similarity" set?

**Comment by rainmanwy** [comment]
yes, could add a keyword for that. Image optiion is suitable for static text.

**Comment by aikarjal** [comment]
I saw that keywords for this were just recently committed to the repo. Great to see this and hopefully we are getting a new version of the library soon!

**Comment by aikarjal** [comment]
@rainmanwy Great, we'll try it out and see how it works. Thanks!

**Comment by aikarjal** [comment]
Our developers told me that they had tested this and it was working.
